documentation/requirements.txt

```diff
@@ -1,3 +1,4 @@
-sphinx==4.1.2
-sphinxcontrib.httpdomain==1.7.0
+sphinx==4.4.0
+sphinxcontrib.httpdomain==1.8.0
 sphinx_fontawesome==0.0.6
+sphinx_copybutton==0.5.0
```

Comment:
Thanks for adding this, I couldn't find this file at first and then forgot about it. Why don't we move this file to our requirements folder, actually? Also, our main `Makefile` doesn't actually read in this requirements file, so either we fix that or we should add the copybutton extension there as well.

Reply:
I merged this already, so I'll change this spearately.